src/react-components/2d-hud.js

```diff
@@ -7,15 +7,21 @@ import styles from "../assets/stylesheets/2d-hud.scss";
 const TwoDHUD = ({ muted, frozen, spacebubble, onToggleMute, onToggleFreeze, onToggleSpaceBubble }) => (
   <div className={styles.container}>
     <div className={cx("ui-interactive", styles.panel, styles.left)}>
-      <div className={cx(styles.iconButton, styles.mute, { [styles.active]: muted })} onClick={onToggleMute} />
+      <div
+        className={cx(styles.iconButton, styles.mute, { [styles.active]: muted })}
+        title={muted ? "Unmute Mic" : "Mute Mic"}
+        onClick={onToggleMute}
+      />
     </div>
     <div
       className={cx("ui-interactive", styles.iconButton, styles.large, styles.freeze, { [styles.active]: frozen })}
+      title={frozen ? "Resume" : "Pause"}
       onClick={onToggleFreeze}
     />
     <div className={cx("ui-interactive", styles.panel, styles.right)}>
       <div
         className={cx(styles.iconButton, styles.bubble, { [styles.active]: spacebubble })}
+        title={spacebubble ? "Bubble Off" : "Bubble On"}
```

Comment:
to be consistent with the other labels, perhaps this should be a verb? unless "bubbling" becomes a verb, heh…

`Enable Bubble`, `Disable Bubble`?

Reply:
Hmm yeah was trying to keep the label fairly short since resizing the `a-rounded` is kind of a pain so I just have it at a static size.. Will try and think of a better word.